pkg/generate/config/operation.go

```diff
@@ -14,11 +14,15 @@
 package config
 
 import (
-	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
-
+	"encoding/json"
 	"github.com/aws-controllers-k8s/code-generator/pkg/util"
+	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
```

Comment:
please change the import section to:
```go
import (
	"encoding/json"

        awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"

	"github.com/aws-controllers-k8s/code-generator/pkg/util"
)
```

Reply:
Sure. `gofmt` does not fix this. Would you recommend using  `goimports` or such to avoid such issues in the future ?

Reply:
Yes exactly :). Sometimes `goimports` doesn't work as expected so you might need to fix the imports order manually. Here are the guidelines for imports https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Thank you!